### PR TITLE
webUI search tests

### PR DESCRIPTION
**Pull request by individual-it**
## Description
tests to perform searches on the webUI

## Related Issue
- needed for https://github.com/owncloud/search_elastic/issues/26

## How Has This Been Tested?
run tests locally, now let :robot: do its job

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/WebUISearchContext.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * ownCloud
+ *
+ * @author Artur Neumann <artur@jankaritech.com>
+ * @copyright Copyright (c) 2017 Artur Neumann artur@jankaritech.com
```
Comment (paurakhsharma) [inline tests/acceptance/features/bootstrap/WebUISearchContext.php]: s/2017/2018

**Review by paurakhsharma — APPROVED** [review]
Looks good

**Review by phil-davis — APPROVED** [review]

**Comment by individual-it** [comment]
backport in #32776

**Comment by patrickjahns** [comment]
Quick note - will there by API tests ? 
It is fine if we have basic acceptance tests on the UI - for more indepth scenario I see API tests more suitable, as the search ( REPORT ) api will be used by desktop/mobile clients as well in the future

**Comment by phil-davis** [comment]
Yes, this test just checks the existing behavior of the current UI search. It will help with getting started on tests for apps with fancier search capability.

**Pull request merged**